Changelog v1.8.x.md

```diff
@@ -3,6 +3,7 @@
 - New: More reinforced block recipes for parity with vanilla (Thanks Redstone_Dubstep!)
 - New: Fake Water/Lava Buckets can now be properly used in dispensers
 - New: Add all ownable blocks to DRAGON_IMMUNE and WITHER_IMMUNE block tags
+- New: Trophy system now destroys Shulker bullets, Dragon fireballs and Wither skulls
```

Comment:
This is a change.

Reply:
>What did you do to the textures to fix the models? (your question below this request)

I noticed (and I noticed it only on the UBM and the UOC, if you think more items have this issue, please tell me them) that you could see through the sides of those tools. This was because there were pixels that were almost invisible due to their high alpha value, but they were there and thus Minecraft rendered the sides of the tools like the most outside pixels of the tool, and as these pixes were in some cases invisible, the outline was, too. All I did was clearing out the invisible pixels from the item texture.